packages/samples/headless-ssr-commerce/app/_components/pages/listing-page.tsx

```diff
@@ -1,29 +1,23 @@
 'use client';
 
-import {NavigatorContext} from '@coveo/headless/ssr-commerce';
-import {useEffect, useState} from 'react';
+import {NavigatorContext} from '@coveo/headless-react/ssr-commerce';
+import {PropsWithChildren, useEffect, useState} from 'react';
 import {
   listingEngineDefinition,
   ListingHydratedState,
   ListingStaticState,
 } from '../../_lib/commerce-engine';
-import BreadcrumbManager from '../breadcrumb-manager';
-import Cart from '../cart';
-import FacetGenerator from '../facets/facet-generator';
-import Pagination from '../pagination';
-import ProductList from '../product-list';
-import {Recommendations} from '../recommendation-list';
-import Sort from '../sort';
-import StandaloneSearchBox from '../standalone-search-box';
-import Summary from '../summary';
+
+interface ListingPageProps {
+  staticState: ListingStaticState;
+  navigatorContext: NavigatorContext;
+}
 
 export default function ListingPage({
   staticState,
   navigatorContext,
-}: {
-  staticState: ListingStaticState;
-  navigatorContext: NavigatorContext;
-}) {
+  children,
+}: PropsWithChildren<ListingPageProps>) {
```

Comment:
We could rename those components (search-page.tsx and listing-page.tsx) to something more like SearchProvider and ListingProvider. What do you think ?